test/lib/browsers/playwright.ts

```diff
@@ -106,6 +106,7 @@ export class Playwright extends BrowserInterface {
       return await chromium.launch({
         devtools: !launchOptions.headless,
         ...launchOptions,
+        ignoreDefaultArgs: ['--disable-back-forward-cache'],
```

Comment:
a very confusing line... but this is saying "don't disable bfcache by default", since Playwright automatically disables it ([ref](https://github.com/microsoft/playwright/blob/576608b69d31b922d356fed6b3efa802e3c42481/packages/playwright-core/src/server/chromium/chromiumSwitches.ts#L26))

Our tests pass seem to pass with it enabled, but I'm not opposed to having this be an option instead